spec/models/preserved_copy_spec.rb

```diff
@@ -194,4 +194,14 @@
       expect(preserved_copy.matches_po_current_version?).to be false
     end
   end
+
+  context '.least_recent_version_audit(last_checked_b4_date, storage_dir)' do
+    let(:db_query) {
+      described_class.least_recent_version_audit(Time.now.utc, 'spec/fixtures/storage_root01/moab_storage_trunk')
+    }
+
+    it 'returns all PreservedCopy objects, in any order' do
+      expect(db_query).to match_array([preserved_copy])
```

Comment:
`db_query` returns `#<ActiveRecord::Relation [#<PreservedCopy id: 1, version: 0, preserved_object_id: 1, endpoint_id: 1, created_at: "2018-01-31 00:33:37", updated_at: "2018-01-31 00:33:37", last_moab_validation: nil, last_checksum_validation: nil, size: 1, status: "validity_unknown", last_version_audit: nil>]>` this is only 1 object, so you can't really test to see if the PreservedCopy object is returned in any order...